packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -21,6 +21,10 @@ import 'text_field.dart';
 import 'theme.dart';
 import 'theme_data.dart';
 
+/// A callback to search which item should be highlighted.
+///
+/// Used by [DropdownMenu.searchCallback].
+typedef SearchCallback<T> = int? Function(List<DropdownMenuEntry<T>> entries, TextEditingController textEditingController);
```

Comment:
Is there a reason to pass along the TextEditingController here? would it be enough to just pass along the controller's text value?

Reply:
Ah Changing it to `String` sounds better. Just updated! Thanks for all the suggestions:) !